1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,7 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+let peopleNames = people.map(element => element.name);
```

Comment:
If you are iterating over an array it is good to use the singular name eg "person" to give more context than "element" or "item"